README.md

```diff
@@ -40,11 +40,11 @@ The libkcapi distribution contains the following applications which are
 intended as a drop-in replacement for the respective applications commonly
 available in all Linux distributions:
 
-* coreutils: sha512sum, sha384sum, sha256sum, sha224sum, sha1sum, md5sum
+* coreutils: sha512sum, sha384sum, sha256sum, sha224sum, sha1sum, md5sum, sm3sum
```

Comment:
AFAIK, the hasher binaries are intended mainly as drop-in replacements for the *sum utilities shipped in coreutils. Do you plan to add `sm3sum` also there? (Or perhaps you/someone already did?)

That said, if you believe providing sm3sum/sm3hmac in libkcapi will be useful to someone, I'm not against adding them.

Reply:
I did implement `sm3sum` and also contributed to coreutils. The community did not accept it, but implemented it in another form `cksum -a sm3`, refer to: https://git.savannah.gnu.org/cgit/coreutils.git/commit/?id=8362f2552a4192d6b1e0e46fa5edd4ce76e29560, coreutils hash algorithm is provided in this form, but for compatibility, the old *sum is still retained.

It is really great to be able to directly provide a `sm3sum`.